crates/bevy_render/src/renderer/render_device.rs

```diff
@@ -2,9 +2,8 @@ use crate::render_resource::{
     BindGroup, BindGroupLayout, Buffer, ComputePipeline, RawRenderPipelineDescriptor,
     RenderPipeline, Sampler, Texture,
 };
-use futures_lite::future;
```

Comment:
I think this is the only `futures_lite` usage in the entire crate. It is probably OK to remove the dependency from the Cargo.toml.

Reply:
there's another
https://github.com/bevyengine/bevy/blob/cdbabb70537b9e0f6b9b9b62de240e7ec637cb01/crates/bevy_render/src/lib.rs#L144-L146